deploy/copy-secrets.sh

```
@@ -0,0 +1,68 @@
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# List of target files
+TARGETS=("cn1" "cn2" "cn3" "agg")
+
+# Sample network private keys
+NETWORK_KEY_CN1="0x11a1e500a548b70d88184a1e042900c0ed6c57f8710bcc35dc8c85fa33d3f580"
+NETWORK_KEY_CN2="0x21a1e500a548b70d88184a1e042900c0ed6c57f8710bcc35dc8c85fa33d3f580"
+NETWORK_KEY_CN3="0x31a1e500a548b70d88184a1e042900c0ed6c57f8710bcc35dc8c85fa33d3f580"
+NETWORK_KEY_AGG="0x41a1e500a548b70d88184a1e042900c0ed6c57f8710bcc35dc8c85fa33d3f580"
+NET_KEYS=($NETWORK_KEY_CN1 $NETWORK_KEY_CN2 $NETWORK_KEY_CN3 $NETWORK_KEY_AGG)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)